wasm/src/programs/manager/transfer.rs

```diff
@@ -113,10 +114,17 @@ impl ProgramManager {
                 ("transfer_private_to_public", inputs)
             }
             "public" | "transfer_public" | "transferPublic" => {
+                let inputs = Array::new_with_length(3);
```

Comment:
This can be better written like this:

```rust
let inputs = [
    JsValue::from(&caller.unwrap()),
    JsValue::from(recipient),
    JsValue::from(&amount_microcredits.to_string().add("u64")),
].into_iter().collect::<js_sys::Array>();
```